config/routes.rb

```diff
@@ -7,6 +7,9 @@
   # route GET /login to login controller and #index action
   get "login", to: "login#index"
   # route GET /check-in to login controller and #index action
-  # get "checkin", to: "checkin#new"
+  get "checkin", to: "checkin#new"
   post "checkin", to: "checkin#create"
+  # the admin dashboard
+  get "admins/index"
```

Comment:
to keep with rails conventions, this should be `get "admin", to: "admin#index"`

also you don't need the 's' at the end once the controller name is also changed.